.github/workflows/ci-schedule.yml

```diff
@@ -0,0 +1,11 @@
+# vim:sw=2:ts=2
+name: ci-schedule
```

Comment:
IMHO don't use `ci` prefix here, name it `nightly-quick` / `nightly-fast` or something like that